src/test/java/liuni/LiUniResourceTests.java

```diff
@@ -0,0 +1,160 @@
+package liuni;
+
+import liuni.api.ErrorModel;
+import liuni.resources.LiUniResource;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import twitter4j.ResponseList;
+import twitter4j.Status;
+import twitter4j.Twitter;
+import twitter4j.TwitterException;
+
+import javax.ws.rs.core.Response;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.reset;
+import static org.mockito.Mockito.when;
+
+public class LiUniResourceTests {
+    @Mock private Twitter twitter;
+    @Mock private Status status;
+
+    @InjectMocks TwitterStatus twitterStatus;
+    @InjectMocks LiUniResource resource;
+
+    @Before
+    public void setUp() {
+        MockitoAnnotations.initMocks(this);
+        twitterStatus.setTwitter(twitter);
+        resource.setTwitterStatus(twitterStatus);
+    }
+
+    @After
+    public void tearDown() {
+        reset(twitter);
+        reset(status);
+    }
+
+    /* Test LiUniResource .fetchTimeline() REST method */
+    @Test
+    public void testREST_fetchTimeline_emptyTimeline() {
+        ResponseList<Status> respList = new ResponseListImpl<Status>();
+        try {
+            doReturn(respList).when(twitter).getHomeTimeline();
+
+            Response resp = resource.fetchTimeline();
+            boolean statusCheck = resp.getStatus() == Response.Status.OK.getStatusCode();
+            boolean entityCheck = resp.hasEntity() && resp.getEntity().equals(respList);
+
+            assertEquals(statusCheck && entityCheck, true);
```

Comment:
split the two up

Reply:
rework you equals and true, etc for the actual individual checking, statusCode for instance.